model/fv_dynamics.F90

```diff
@@ -630,26 +636,16 @@ subroutine fv_dynamics(npx, npy, npz, nq_tot,  ng, bdt, consv_te, fill,
                                           reg_bc_update_time )
          endif
 #endif
-
-         if( last_step )  then
-            if( .not. hydrostatic ) then
-!$OMP parallel do default(none) shared(is,ie,js,je,npz,omga,delp,delz,w)
-               do k=1,npz
-                  do j=js,je
-                     do i=is,ie
-                        omga(i,j,k) = delp(i,j,k)/delz(i,j,k)*w(i,j,k)
-                     enddo
-                  enddo
-               enddo
-            endif
```

Comment:
@lharris4 - is this still needed?

Reply:
No, because we now use the correct definition of omega in non-hydrostatic simulation, which is the same as in the hydrostatic model.